Test/com/nicoles_change_maker/test/ChangeMakerTest.java

```diff
@@ -0,0 +1,235 @@
+package com.nicoles_change_maker.test;
+import com.nicoles_change_maker.ChangeMaker;
+import org.junit.*;
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by bears8yourface on 1/5/16.
+ */
+
+public class ChangeMakerTest {
+    ChangeMaker changeMaker;
+    HashMap expectedChange = new HashMap<String,Integer>();
```

Comment:
I get a compiler warning on this line and every line below that uses this `HashMap`:

<img width="1440" alt="generics" src="https://cloud.githubusercontent.com/assets/1131633/12399565/c802c690-bde8-11e5-95d1-2c63897df4a7.png">

I know this code was from your First Day of Java Ever, but you're probably ready to read up on Generics, which are the source of this warning: https://docs.oracle.com/javase/tutorial/java/generics/index.html. Chapter 16 in Head First Java covers Generics, too.

The fix here is moving your type parameters to the other side of the equals sign:

```java
HashMap<String, Integer> expectedChange = new HashMap<>();
```

...but I'll let you read the docs to understand why.